auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -623,7 +572,8 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
     color: $app-blue !important;
   }
 
-  .v-list .v-list-item--active, .v-list .v-list-item--active {
+  .v-list .v-list-item--active,
+  .v-list .v-list-item--active {
```

Comment:
I like this change -- much easier to see that it's separate selectors with the same styles.